dd-trace-ot/src/main/java/datadog/opentracing/DDSpanContext.java

```diff
@@ -321,6 +326,25 @@ public synchronized void setTag(final String tag, final Object value) {
     return Collections.unmodifiableMap(tags);
   }
 
+  /**
+   * Add a logged event map to the span. Logs are not propagated to the children.
+   *
+   * @param l the timestamp of the event.
+   * @param map the event map for the tag.
+   */
+  public synchronized void log(final long l, final Map<String, ?> map) {
+    List<Map<String, ?>> logItems = logs.get(l);
+    if (logItems == null) {
```

Comment:
Since we're in Java 8, you can replace the body of this method with:

```java
logs.computeIfAbsent(timestamp, ArrayList::new).add(map);
```

Reply:
I think the source compatibility for the project is 7: https://github.com/signalfx/signalfx-java-tracing/blob/master/gradle/java.gradle#L14

Reply:
Sigh, ok. Java 7 is EOL already, but not worth the effort here